src/tests/nativeaot/SmokeTests/PInvoke/PInvokeNative.cpp

```diff
@@ -624,9 +624,9 @@ DLL_EXPORT bool __stdcall VerifyAnsiCharArrayIn(char *a)
 
 DLL_EXPORT bool __stdcall VerifyAnsiCharArrayOut(char *a)
 {
-    if (a == NULL) 
+    if (a == NULL)
         return false;
-    
```

Comment:
Apologize for noise like that. Should I revert these changes? Probable VS or VS code mess up

Reply:
This is ok. It is caused by `.editorconfig` in the root.